src/routes/acquirerRoutes.ts

```diff
@@ -53,8 +54,20 @@ const skipThisStep = (step: string, formdata: FormData) => {
       const legalGatewayStep = formdata.steps["legal-gateway"].value as LegalGatewayStep
       return legalGatewayStep.yes.checked || legalGatewayStep.other.checked
     }
+    case "lawful-basis-personal": {
+      // Skip lawful-basis-personal if answer to data-type was "special" or "none"
+      const DataTypeStep = formdata.steps["data-type"].value as DataTypeStep
+      return DataTypeStep.special.checked || DataTypeStep.none.checked
```

Comment:
I think special can be checked it will not affect lawful-basis-personal, what will affect is personal to be false. If personal is true and special is true, it will still work